### DISCUSSION TITLE
Helle Guys

### DISCUSSION THREAD

**Discussion by danielmarv** [community/community#52327] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Am just new to this Github community So Big up

**Comment by Akash1134** [comment, marked as answer]
👋 Welcome to the GitHub Community on Discussions, @danielmarv  Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).

**Reply by danielmarv** [reply]
DEVOPS

**Reply by 8117068471** [reply]
![901AB569-4BB1-44CC-8378-8ECA81749867](https://github.com/community/community/assets/170523773/207633a1-6a97-4478-9985-756cc4185123)
